### PR TITLE
Add material ui

### PR DESCRIPTION
**Pull request by koltclassic**
This PR adds material ui and sets up the bottom navigation.
For some reason the default material-ui dark theme colors feel a bit off contrast-wise... I'll look into that.
Dec-01-2020 20-03-16

### CODE REVIEW

File: components/BottomNavigationActionLink.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { BottomNavigationActionProps, Button, LinkProps } from "@material-ui/core"
+import React from "react"
+import Link from "next/link"
+
+export type BottomNavigationActionLinkProps = Omit<BottomNavigationActionProps, 'href' | 'classes'> &
```
Comment (koltclassic) [inline components/BottomNavigationActionLink.tsx]: I had to use this workaround for the material-ui BottomNavigationAction and nextjs Link component interaction as BottomNavigationAction labels weren't appearing using more conventional methods. The types seem to be a bit off here.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,5842 @@
+{
```
Comment (GuthrieW) [inline package-lock.json]: Can we get rid of this file and add the stuff you need with yarn instead?

File: components/BottomNavigationActionLink.tsx (hunk 1 above)
Comment (GuthrieW) [reply components/BottomNavigationActionLink.tsx]: As long as it works I'm happy!

File: pages/Community/index.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```
Comment (GuthrieW) [inline pages/Community/index.tsx]: Not something I'm going to not approve any commits for but there's a .pretterrc file all set up that should take care of code styling for us. Just noticed there was a semi colon here.

**Review by GuthrieW — APPROVED** [review]

**Comment by GuthrieW** [comment]
Looks great man!

**Pull request merged**